rundeckapp/grails-app/init/rundeckapp/Application.groovy

```diff
@@ -65,6 +61,17 @@ class Application extends GrailsAutoConfiguration implements EnvironmentAware {
         loadGroovyRundeckConfigIfExists(environment)
     }
 
+    @Override
+    void doWithApplicationContext() {
+        if(rundeckConfig.isRollback()) {
+            RundeckDbMigration rundeckDbMigration = new RundeckDbMigration(applicationContext, grailsApplication)
+            rundeckDbMigration.rollback(migrationsDir, rundeckConfig.tagName())
```

Comment:
There does not appear to be a need for the migrationsDir var passed to this method.